deploy/values-dev.yaml

```diff
@@ -0,0 +1,53 @@
+applicationName: "inventory"
+deployment:
+  annotations:
+  podLabels:
+    app: inventory
+  volumes: {}
+  image:
+    repository: stakater/stakater-nordmart-inventory
+    tag: v0.0.3
```

Comment:
@kahootali will we hard code the tag?

Reply:
Doesn't matter, what value we set here, it will be overriden in pipeline by using `--set`